hypertrace-core-graphql-attribute-store/src/main/java/org/hypertrace/core/graphql/attributes/AttributeClient.java

```diff
@@ -32,6 +33,7 @@
       AttributeModelTranslator translator) {
     this.grpcContextBuilder = grpcContextBuilder;
     this.translator = translator;
+    this.serviceConfig = serviceConfig;
```

Comment:
Can you please add unit test for this?

Reply:
Hi Jayesh, sure I can add a UT for this (was even thinking of doing it before raising the PR). However, @aaron-steinfeld had [this](https://github.com/hypertrace/hypertrace-graphql/pull/91#issuecomment-877638416) to say about adding UTs to these. There isn't much I can test and assert on here (except that the test config is read correctly). If you still think a UT is required, I'll gladly add it :)

Reply:
Got it! agreed!